lib/sinon/util/fake_xml_http_request.js

```diff
@@ -611,4 +611,4 @@
         makeApi(sinon);
     }
 
-})(typeof self !== "undefined" ? self : this);
+})(typeof global !== 'undefined' ? global : this);
```

Comment:
@cjohansen The build is failing on master because there are single quotes here instead of the jscs required double quotes.